packages/plugin-loans-api/src/models/utils/scheduleUtils.ts

```
@@ -165,92 +166,135 @@ export const scheduleHelper = async (
   const skipInterestCalcDate = getSkipDate(currentDate, skipInterestCalcMonth, skipInterestCalcDay);
   const skipAmountCalcDate = getSkipDate(currentDate, skipAmountCalcMonth, skipAmountCalcDay);
 
-  if (repayment === 'equal') {
-    const payment = new BigNumber(balance - (salvageAmount || 0)).div(paymentDates.length).dp(calculationFixed, BigNumber.ROUND_HALF_UP).toNumber()
+  switch (repayment) {
+    case 'equal':
+      const payment = new BigNumber(balance - (salvageAmount || 0)).div(paymentDates.length).dp(calculationFixed, BigNumber.ROUND_HALF_UP).toNumber()
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)